src/components/FAQ/FAQ.jsx

```diff
@@ -16,6 +16,16 @@ const useStyles = createStyles((theme) => ({
 
     control: {
         color: 'white',
+        transition: 'background-color 250ms ease',
```

Comment:
Transition is not working because background-color targets a different property.